redocly.yaml

```diff
@@ -0,0 +1,53 @@
+theme:
```

Comment:
Would be better to put all files related to redocly to `readocly` folder.

Reply:
Good idea thank you!

Reply:
@cszhu Did you forget to put `readocly.yaml` in `readocly` folder?

Reply:
Yes I forgot, just fixed that !! Thank you for catching it